README.md

```diff
@@ -67,13 +67,15 @@ Download the latest version of PrebidJS
 ```
 $ git clone https://github.com/prebid/Prebid.js.git
 $ cd Prebid.js
+$ git checkout 1.6.0 # optional
```

Comment:
Should this be flagged as optional?

Reply:
Hm, up to you guys I think. Seems preferable to build a specific version rather than the bleeding edge, but either should work